packages/bpk-component-bottom-sheet/src/__snapshots__/BpkBottomSheet-test.tsx.snap

```diff
@@ -52,7 +52,7 @@ exports[`BpkBottomSheet renders correctly with action props 1`] = `
               class="bpk-navigation-bar__title bpk-navigation-bar__title--default"
             >
               <span
-                class="bpk-text bpk-text--heading-5"
+                class="bpk-text bpk-text--label-1"
```

Comment:
Interesting these have now changed as this could lead me to believe these used to be wrong!

Reply:
Indeed!

Reply:
Although since the span was empty it wouldn't have been a noticeable difference either way.